examples/widgets/pickPositionWidget.js

```diff
@@ -0,0 +1,37 @@
+/* global itowns */
+/* global Widget */
+/* global widgetTab */
```

Comment:
remove global  widgetTab 

User must do widgetTab.add(pickPositionWidget);

Do example to test pickPositionWidget